script.js

```diff
@@ -0,0 +1,52 @@
+
+//	Ensure javascript is loaded after html/css
+$(document).ready(function() {
+	$('form').submit(function(event){
+		event.preventDefault();
+
+//	Establish variable for storage of entries
+
+		var city = $("#city-type").val();
+		var city = city.toLowerCase();
```

Comment:
you shouldn't double declare the same variable.

Declaration is this:
```js
var city;
``` 
It basically just says, he program, this variable exists now.
Assignment is this:
```js
city = 'something'; // it could be anything on the right hand side of the equal sign
```
This says, hey city variable that I already declared, I want you to be this value.

You can do both declaration and assignment in one line like so:
```js
var city = 'something';
```
That both declares the variable (`var city`) and assigns it (`city = something;`).

Once a variable is declared and assigned, you can *reassign* it just like a normal assignment.

```js
var city = 'something';

// this is the REassignment
city = 'something else';
```

Reassignments are just like assignments. Its simply `variableName = valueYouWantSetToTheVariable`.

But note that when reassigning a variable, you do not put `var` in front of it again. `var` only happens once: the very first time you declare a variable.

Also, FWIW, you must always *declare* a variable before *assigning* a variable.